### PR TITLE
Week 8 Portfolio - Joanna Lodell

### PR DESCRIPTION
**Pull request by joannalodell19**
View it live here: https://gallant-sammet-84d996.netlify.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+        </div>
+    </header>
+
+    <!-- INTRO -->
+    <section class="section">
+        <div class="container">
+            <div class="intro-text">
+                <p>
+                <span class="line"></span>
+                <span class="dot"></span>
```
Comment (vanhaj) [inline index.html:49]: Really cool way to use the span here instead of inserting a picture!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">    
+    
+    <link rel="stylesheet" href="./style.css" />
+    <title>Joanna Lodell, Portfolio</title>
+</head>
+
+<body>
+    <!-- HEADER -->
+    <header class="header">
+        <div class="container header-container">
```
Comment (vanhaj) [inline index.html:20]: Really nice to see that you are reusing classes and also adding new classes on the sam tag!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+    <!-- MY THOUGHTS -->
+    <section class="section">
+        <div class="container thoughts-section">
+            <h2 class="section-heading thoughts-heading">MY THOUGHTS</h2>
+            <div class="thoughts-posts">
+                <div class="thoughts-group">
+                    <img class="thoughts-image" src="./assets/thoughts-image.jpg" alt="Image displaying computer on desktop.">
+                    <div class="thoughts-text-group">
+                        <h5>FEB 2022</h5>
+                        <h6>How to stop being scared: <br> Learning to love code</h6>
```
Comment (vanhaj) [inline index.html:342]: You are doing a great job with using the headings!

File: index.html [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+                    <p class="skills">React</p>
+                    <p class="skills">Redux</p>
+                    <p class="skills">Node.js</p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- FOR MORE -->
+    <section class="section">
```
Comment (vanhaj) [inline index.html]: I apperciate that you put everything in each section, that makes it also easier to follow your code.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,279 @@
...
+        </div>
+    </section>
+
+    <!-- FOOTER -->
+    <footer class="footer">
+        <div class="container contact-section">
+            <h2 class="contact-heading">CONTACT</h2>
+            <p>Joanna Lodell</p>
+            <p>+46 704 18 48 50</p>
+            <p class="email">joanna@pepped.se</p>
```
Comment (vanhaj) [inline index.html]: I have a tip for you! To make the email clickable and directs it straight to writing an email, you could write your code like this instead:
```
<p class="email">
          <a href="mailto:joanna@pepped.se">joanna@pepped.se</a>
 </p>
```

File: style.css [hunk 6]
```diff
@@ -0,0 +1,609 @@
...
+/* Global styles */
+.container {
+    width: 375px;
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+/* HEADER */
```
Comment (vanhaj) [inline style.css:16]: The commets makes is super clear what the code is used for!

File: style.css [hunk 7]
```diff
@@ -0,0 +1,609 @@
...
+    padding-top: 58px;
+    font-family: Montserrat;
+    font-weight: 700;
+    font-style: normal;
+    font-size: 21px;
+    line-height: 25px;
+    text-align: center;
+}
+
+.more-thoughts-group, .thoughts-link, .more-thoughts-first-link {
```
Comment (vanhaj) [inline style.css:351]: Awsome method to use classes for the same purpose!

File: style.css [hunk 8]
```diff
@@ -0,0 +1,609 @@
...
+	}
+}
+
+/* TABLET */
+@media screen and (min-width: 768px) and (max-width: 1023px){
+	.container {
+		width: 60vw;
+        margin: 0 auto;
+
+	}
```
Comment (vanhaj) [inline style.css:513]: Just a small suggestion, tab in the width property and display: none on line 488 to make it look "prettier"

**Review by vanhaj — APPROVED** [review]
Great job Joanna!

I really liked that you took the time and linked to each project and the same with the icons.
Some of the sections in mobile view seemd to need a little more spacing on the sides.
Other than that, I have nothing more to add! Your webpage is looking really clean and is very easy to follow. I also enjoyed your background picture in the header, looked so dreamy!

Thank you for sharing your code :)

File: index.html (hunk 5 above)
Comment (joannalodell19) [reply index.html]: @vanhaj - Ah, that's great! I knew that but didn't even think to implement it here. Thanks!

File: style.css (hunk 8 above)
Comment (joannalodell19) [reply style.css:513]: @vanhaj Big thanks!

**Pull request closed without merging**